apps/tangle-cloud/src/pages/instances/Instances/UpdateBlueprintModel/ApproveConfirmationModal.tsx

```diff
@@ -86,35 +112,37 @@ function ApproveConfirmationModal({
       description="Are you sure you want to approve this blueprint?"
     >
       <ModalHeader onClose={onClose}>
-        Service Request #{selectedRequest?.requestId}
+        Service Request #{selectedRequest?.requestId?.toString()}
```

Comment:
If `selectedRequest` or `requestId` is `undefined` or `null`, the expression evaluates to `undefined`, and the rendered text will be:

Service Request # (with nothing after the #)